.github/workflows/sync-spec-test-suite.yml

```diff
@@ -0,0 +1,83 @@
+name: Sync spec test suite from NodeJS toolbox
+on:
+  pull_request:
+  schedule:
+    # https://crontab.guru/every-night-at-midnight
+    - cron: "0 0 * * *"
+
+jobs:
+  sync-spec-test-suite:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
+          extensions: dom, iconv, json, libxml, zip
+          tools: composer
+
+      - name: Get Composer Cache Directory
+        id: composer-cache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Configure Composer cache
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Install Composer dependencies
+        run: composer install
+
+      - name: Configure git user
+        run: |
+          git config --global user.email "alain.schlesser@gmail.com"
+          git config --global user.name "Alain Schlesser"
+
+      - name: Create branch to base pull request on
+        run: |
+          git fetch
+          git branch -f sync-spec-test-suite origin/main
+          git checkout sync-spec-test-suite
+          git pull --no-rebase || true
+
+      - name: Sync spec test suite
+        run: composer sync-test-specs
+
+      - name: Check if there are changes
+        id: changes
+        uses: UnicornGlobal/has-changes-action@v1.0.11
+
+      - name: Get pull request body text
+        if: steps.changes.outputs.changed == 1
+        id: body
+        run: |
+          BODY=$(bin/render-pr-body.bash)
+          echo $BODY
+          BODY="${BODY//$'\n'/'%0A'}"
+          BODY="${BODY//$'`'/'\`'}"
+          echo -e "::set-output name=text::$BODY"
+
+      - name: Commit changes
+        if: steps.changes.outputs.changed == 1
+        run: |
+          git add --all .
+          git commit -m "Sync spec test suite - $(date +'%Y-%m-%d')"
+          git push origin sync-spec-test-suite
+
+      - name: Create pull request
+        if: steps.changes.outputs.changed == 1
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: "sync-spec-test-suite"
+          destination_branch: "main"
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          pr_title: "Sync spec test suite"
```

Comment:
Was a date provided originally here? I see it here: https://github.com/ampproject/amp-toolbox-php/pull/57